index.html

```diff
@@ -183,6 +183,7 @@ <h2 class="title-contact">Contact me</h2>
             <textarea name="message" id="message" cols="30" rows="10" placeholder="Write your message here" maxlength="500"
                 required></textarea>
             <button type="submit"id="contact-button">Get in touch</button>
+           <p class="separator"></p>
         </form>
        
     </section>
```

Comment:
- [ ] Kindly 😃 as stated by the previous reviewer In addition to the feedback on the form, I noticed that there is a horizontal scrolling issue on your portfolio. To improve the user experience, it would be better to address this issue, so please try to fix that to be like the Figma Design 😊

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Your  Form |
| --- | 
|   ![Recording 2023-03-02 at 17 04 19 (1)](https://user-images.githubusercontent.com/83360636/222466647-0fe62c7a-4af8-4123-89e4-3bae2ec80cf4.gif)  |